packages/office-ui-fabric-react/src/components/SearchBox/examples/SearchBox.CustomIcon.Example.tsx

```diff
@@ -1,12 +1,18 @@
 import * as React from 'react';
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
-import './SearchBox.Examples.scss';
+import { mergeStyleSets } from '@uifabric/styling';
+
+const styles = mergeStyleSets({
```

Comment:
Since this example just has one SearchBox, it doesn't need any extra styles--the margin style was just added to make space between SearchBoxes.

Reply:
Understood! Removed!